libs/protos/protos/steps/sp_steps_custom.proto

```diff
@@ -4,7 +4,6 @@ package protos.steps;
 
 option go_package = "github.com/streamdal/streamdal/libs/protos/build/go/protos/steps";
 
-// WIP -- Custom WASM exec?
 message CustomStep {
-  string id = 1;
+  map<string, bytes> args = 1;
```

Comment:
Don't think we need bytes here. They will be included in `SetPipelinesCommand` and `GetSetPipelinesCommandsByService` messages

Reply:
I was thinking of it as an `interface{}`-like type, but I also just realized that map is dumb as it won't be ordered.

It should be `repeated bytes`, no? Having args as `repeated string` seems incorrect.